sdk/communication/communication-sms/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azure/communication-sms",
-  "version": "1.1.0",
+  "version": "1.1.0-beta.1",
```

Comment:
Is this intended?

Reply:
Yeah the next version in the Changelog is `1.1.0-beta.1`. The miss-match was causing an error in Analyze :(

There seems to be another PR trying to fix this: https://github.com/Azure/azure-sdk-for-js/pull/21345/files